contract/r/gnoswap/gov/governance/vote_test.gno

```diff
@@ -182,7 +183,14 @@ func TestVote(t *testing.T) {
 			func() {
 				defer func() {
 					if r := recover(); r != nil {
-						err = r.(error)
+						switch v := r.(type) {
```

Comment:
Could you let me know why thus logic was added?

Reply:
@kjeom This change was to handle a type error in the `error` type by ensuring it's checked properly. Therefore, this change is necessary.